staging/src/k8s.io/legacy-cloud-providers/aws/aws_test.go

```diff
@@ -1674,10 +1675,11 @@ func TestLBSecurityGroupsAnnotation(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
```

Comment:
IMO we need to define the behavior we expect using tests. We need a much more exhaustive set of tests for the various cases and combinations. For example:

- "service.beta.kubernetes.io/aws-load-balancer-security-groups" takes precedence over global
- a security group is created iff neither "service.beta.kubernetes.io/aws-load-balancer-security-groups" nor global exist
- "service.beta.kubernetes.io/aws-load-balancer-extra-security-groups" is always added
- no security groups are ever marked for setup or modifiable EXCEPT the kubernetes-created one

Reply:
So You want me to adjust the test cases (or add some) that they meet these requirements?

Reply:
If you don't mind. Or I could add them myself in a follow-up, I don't want it to block the PR